fighthealthinsurance/ml/ml_router.py

```
@@ -695,27 +712,57 @@ async def summarize(
+                f"Summarize the following {title} for use in a health insurance "
+                f"appeal: {abstract_optional}. {instructions}"
             )
-            if r is not None:
-                return r
+        for m in models:
+            for prompt in attempts:
+                try:
+                    r = await m._infer_no_context(
+                        system_prompts=system_prompts, prompt=prompt
+                    )
+                except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)